.github/workflows/gating.yaml

```diff
@@ -101,3 +109,44 @@ jobs:
         ignore: DL3041
         failure-threshold: warning
 
+  build-image:
+    name: Build Cachi2 image and run smoketest on it
+    runs-on: ubuntu-latest
+    steps:
+     # Authenticate to container image registry to push the image
+      - name: Podman Login
+        uses: redhat-actions/podman-login@v1.4
+        with:
+          registry: ${{ env.IMAGE_REGISTRY }}
+          username: ${{ secrets.QUAY_USER }}
```

Comment:
@chmeliik  @taylormadore  @brunoapimentel  @ejegrova  Anyone knows why I cannot pass repository secrets to GitHub Workflow? :(
* https://github.com/containerbuildsystem/cachi2/actions/runs/3419507522/jobs/5693115442
They were added here:
* https://github.com/containerbuildsystem/cachi2/settings/secrets/actions
Is there any hidden flag to make them "more" public in settings? So far did not find anything.

Reply:
Could it be that every fork needs its own secrets, same as gitlab?

Reply:
Now that I'm looking at operator-manifest, it seems that the multiarch-build workflow only happens on merges to master: https://github.com/containerbuildsystem/operator-manifest/actions

Which is what we want, right? We shouldn't be pushing to quay.io for every PR. So it should be OK to have the credentials only in this repo

Reply:
yes that must be it. I'll add them to my fork to test pushing content and then I will change it to do it only on merges to main/master